dcos/http/transport/README.md

```diff
@@ -4,6 +4,6 @@
 - package breaks `RoundTripper` interface spec defined in `https://golang.org/pkg/net/http/#RoundTripper`
   by mutating request instance.
 
-jwt/transport is an http.RoundTripper implementation that automatically adds `Authorization` header to each request with
-a signed token. If request returns 401 response code, the library will generate a new token, sign it with a bouncer and
-retry the current request.
+jwt/transport is an http.RoundTripper implementation that automatically adds `User-Agent` and `Authorization` headers
+to each request with a signed token. If request returns 401 response code, the library will generate a new token, sign
+it with a bouncer and retry the current request.
```

Comment:
This makes it sound like the `User-Agent` header also includes a token. Something like this would be clearer:

```
jwt/transport is an http.RoundTripper implementation that adds `Authorization` and `User-Agent` headers to each request. The `Authorization` header is a signed Javascript web token (JWT). The `User-Agent` defaults to `dcos-go`, and may be customized. If a request returns a 401 response code, the library will update the `Authorization` header with a new signed token and retry the request.
```